addons/godot-firebase/auth/auth.gd

```diff
@@ -460,19 +460,20 @@ func _on_FirebaseAuth_request_completed(result : int, response_code : int, heade
 
 # Function used to save the auth data provided by Firebase into an encrypted file
 # Note this does not work in HTML5 or UWP
-func save_auth(auth : Dictionary) -> void:
+func save_auth(auth : Dictionary) -> bool:
     var encrypted_file = FileAccess.open_encrypted_with_pass("user://user.auth", FileAccess.WRITE, _config.apiKey)
     var err = encrypted_file == null
     if err:
         Firebase._printerr("Error Opening File. Error Code: " + str(FileAccess.get_open_error()))
     else:
         encrypted_file.store_line(JSON.stringify(auth))
+    return not err
```

Comment:
Do we _know_ this works in 4.0? I ask because they changes the parser so it no longer accepts stuff like "not object". I don't think err counts as an object here, but we should be certain.

Reply:
Yes it works indeed!
At this point `err` is not an object but just a boolean (`encrypted_file == null` -> true/false)

Reply:
Ohh, I see. I misread that line, looks good!